Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -101,7 +106,16 @@ define([
         }
         //>>includeEnd('debug');
 
+        if (defined(options.times) && !defined(options.clock)) {
+            throw new DeveloperError('options.times was specified, so options.clock is required.');
+        }
+
+        if (defined(options.proxy)) {
+            deprecationWarning('WebMapServiceImageryProvider.proxy', 'The options.proxy parameter has been deprecated. Specify options.url as a Resource instance and set the proxy property there.');
```

Comment:
This deprecation was removed in the master for our release next week. `deprecationWarning` isn't included in this file anymore.